samples/quickstart.js

```diff
@@ -16,7 +16,7 @@
 'use strict';
 
 // [START translate_quickstart]
```

Comment:
The region tag here should be moved right before the `// Imports the Google Cloud...`  Inside of the `main` function, after we instantiate the client, create an `async quickstart` method that takes no params.  Then call it!